tests/storage/datavolume.go

```diff
@@ -303,7 +303,7 @@ var _ = SIGDescribe("DataVolume Integration", func() {
 				}
 			})
 
-			It("[test_id:3189]should be successfully started and stopped multiple times", func() {
+			It("[test_id:3189]should be successfully started and stopped multiple times", decorators.Conformance, func() {
 				sc, exists := libstorage.GetRWOFileSystemStorageClass()
 				if !exists {
 					Skip("Skip test when Filesystem storage is not present")
```

Comment:
Thanks, I wasn't aware of these. I fixed that specific occurrence, but I'm afraid some others have skips embedded in them too https://github.com/kubevirt/kubevirt/blob/ba06da311be77f2eec3ddd4a018b9df739ca540f/tests/storage/export.go#L492-L498 which is a bummer, since I believe this is desired in the conformance.

I think that for sig-storage, we would have to tidy up the requirements. sig-storage-conformance makes no sense without block rwx and rwo for fs/block.

Reply:
> sig-storage-conformance makes no sense without block rwx and rwo for fs/block.

I think that the skips should be converted to decorator such as `RequiresFilesystemStorageClass`. Then, someone that want to run storage-conformance on a cluster without fs would need to add `!filesystem-storage-class`.

If we keep the skip, someone may accidentally run Conformance on a cluster without filesystem, pass it (with skips) and would be unaware that they have a bug.

Reply:
> I think that the skips should be converted to decorator such as `RequiresFilesystemStorageClass`. Then, someone that want to run storage-conformance on a cluster without fs would need to add `!filesystem-storage-class`.

Yes, that's totally the right way to do things; I think it could even be simpler for RWO FS since it's consensus (see the second commit) so the label could be avoided. However, I prefer to tackle the skips surgically for the proposed conformance set at this time. Hopefully others in sig-storage who own these suites help tidy skips up further.

Reply:
> However, I prefer to tackle the skips surgically for the proposed conformance set at this time

It looks like these skips are too intertwined in the code base.. I may have to sideline this PR while I tackle them all/most. Does that make sense?

Reply:
It makes sense to me, but I hope you can start by introducing all the skipless tests into `Conformance` and continue with the others as they are cleaned.